webapp/components/wpt-results.js

```diff
@@ -1091,6 +1096,23 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
     this.handleSubmitQuery();
     this.dismissToast(e);
   }
+
+  computeResultsTotalsRangeMessage(searchResults, shas, productSpecs, from, to, maxCount, labels, master) {
+    const msg = super.computeResultsRangeMessage(shas, productSpecs, from, to, maxCount, labels, master);
+    if (searchResults) {
+      let subtests = 0, tests = 0;
+      for (const r of searchResults) {
+        if (r.test.startsWith(this.path)) {
+          tests++;
+          subtests += Math.max(...r.legacy_status.map(s => s.total));
+        }
+      }
+      return msg.replace(
+        'Showing ',
+        `Showing ${tests} tests (${subtests} subtests) from `);
```

Comment:
In https://totals-dot-wptdashboard-staging.appspot.com/results/?label=master&product=chrome%5Bexperimental%5D&product=edge&product=firefox%5Bexperimental%5D&product=safari%5Bexperimental%5D&aligned&q=chrome%3Aerror I don't see this message. Is something not wired up correctly?